docs/source/contributing/version-policy.md

```diff
@@ -57,14 +57,6 @@ Python 3.8 support was dropped in 6.0.14.
 
 On Plone 6, we recommend using the known good set (KGS) of package versions that are specified in the Plone release.
 
-On Plone 5, Volto is currently tested with the following packages pinned to specific versions, and we recommend using the same versions, which are:
-
--   `plone.restapi` 9.2.0
--   `plone.rest` 3.0.1
--   `plone.volto` 4.1.0
-
-For the Plone 5 series, the [latest released version of Plone 5](https://plone.org/download/releases) is recommended.
-
 Volto should work on older Plone versions as well, since Plone REST API supports Plone back to version 4.3 (including Archetypes).
```

Comment:
We should probably change this paragraph to say something like Volto isn't tested with older Plone versions.

And remove 5.2 from the compatibility table above.

Reply:
Done